trace/stats.go

```diff
@@ -141,37 +142,30 @@ func (s *PacketCounter) Summary(n int) *PacketCountSummary {
 	}
 }
 
-func topNByTcpPacketCount[T comparable](counts map[T]*PacketCounts, n int) map[T]*PacketCounts {
-	if n <= 0 {
-		return map[T]*PacketCounts{}
-	}
+type pair[T constraints.Ordered] struct {
+	k T
+	v *PacketCounts
+}
```

Comment:
I wondered if this could be put inside the function, but it cannot: "type declarations inside generic functions are not currently supported"?!

Reply:
I know!  I tried that first, but no luck 😞